package.json

```diff
@@ -1,11 +1,13 @@
 {
   "name": "ixbrl-viewer",
-  "version": "0.0.0",
   "description": "iXBRL Viewer",
   "main": "iXBRLViewerPlugin/viewer/src/js/ixbrlviewer.js",
   "directories": {
     "test": "tests"
   },
+  "files": [
+    "iXBRLViewerPlugin/viewer"
+  ],
```

Comment:
@austinmatherne-wk @paulwarren-wk 
I wouldn't think anything outside of the `/viewer` folder would need to be included in the package (besides the package config/readme included by default). I didn't go down to `/dist` because others may want to consume the `/src` JS, which w-ixbrl-viewer currently does via import (which uses the `main` attribute above)

Reply:
I think this is reasonable for the future, but I suspect that right now it will break the w-ixbrl-viewer build as it depends on pulling in the Python code through the NPM install